### PR TITLE
Rework release team lead handbook & init release team lead progress issue

### PR DESCRIPTION
**Pull request by leonardpahlke**
Signed-off-by: leonardpahlke <leonard.pahlke@googlemail.com>

#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

The current Release Team Leads handbook mixes action items with guidance. Due to that following up action items and filtering out guidance is more difficult. At the same time, we are using a ["Release Team Lead onboarding"](https://github.com/kubernetes/sig-release/issues/new?assignees=&labels=sig%2Frelease%2C+area%2Frelease-eng%2C+area%2Frelease-team&template=release-team-lead.md&title=Release+Team+Lead+access+for+%3CGH-handle%3E) issue with success, I would say.

This PR attempts to move action items into a new issue template, which gets merged with the existing "Release Team Lead onboarding" issue-template. This should first of all simplify the work for the Release Team Lead and also increase transparency (everyone can see which tasks are already done).

Also, the Release Team Leads manual currently lists explicit weeks that don't apply to all releases, so this PR uses phases like: `1. Before the start of the Release Cycle (Week 0)` / `2. First weeks of the release cycle up to Enhancements Freeze (1-3 Weeks)`
 
#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

Since this is quite a big change a lot of eyes on this would be good!

previous release team leads and ea's
/cc @palnabarun @cici37 @reylejano @JamesLaverack @jrsapi @jeremyrickard 
cc @kubernetes/sig-release-leads & @kubernetes/release-team-leads

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:30]:
```suggestion
- [ ] Release directory named `release-1.XX` added to [k/sig-release/releases](https://github.com/kubernetes/sig-release/tree/master/releases)
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:85]:
```suggestion
  - Send the email to [k-dev](https://groups.google.com/a/kubernetes.io/g/dev), [SIG Leads](https://groups.google.com/a/kubernetes.io/g/leads), [SIG Release](https://groups.google.com/forum/#!forum/kubernetes-sig-release), and [Release Team](https://groups.google.com/a/kubernetes.io/g/release-team)
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:86]: These examples are on the release-team mailing list, so they don't work if you're not already on the team. Change to the k-dev versions?

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:121]:
```suggestion
- [ ] Remind the community about Enhancements Freeze reminder send out. Example email: [1.26](https://groups.google.com/a/kubernetes.io/g/dev/c/pJZC2cnkeJ8); It may also be useful to send a short version to the `#sig-release` and `#chairs-and-techleads` Slack channels referencing to the k-dev email.
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:135]:
```suggestion
- [ ] Checked in with ci-signal and branch managers if 1.XX.0-alpha.3 is ok to be released and [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:136]:
```suggestion
- [ ] Checked in with Release Notes team on release notes progress after the release notes for 1.XX.0-alpha.3 is generated
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:139]:
```suggestion
- [ ] Checked in with ci-signal and branch managers if 1.XX.0-alpha.4 is ok to be released and [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:140]:
```suggestion
- [ ] Checked in with Release Notes team on release notes progress after the release notes for 1.XX.0-alpha.4 is generated
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:149]:
```suggestion
- [ ] The first retrospective meeting is scheduled for the first week of Monday, Wednesday, and Friday burndown meetings, typically mid-week. Confirm the Emeritus Adviser can serve as facilitator. If Emeritus Adviser is unavailable then defer the responsibility as appropriate.
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:151]:
```suggestion
- [ ] Started release team meetings on Monday, Wednesday, and Friday
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:171]:
```suggestion
- Monitor the enhancements GitHub project to get an idea of how many PRs are still outstanding leading up to Code Freeze
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:186]:
```suggestion
- [ ] Verified that the [`release-1.XX` branch](https://github.com/kubernetes/kubernetes/branches) has been automatically created at the start of Test Freeze
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:201]:
```suggestion
- [ ] The task is now to ensure the release branch is ready to go. This means there are zero pending PRs, no failing 1.XX-blocking tests, no open issues in the milestone. This will continue until release day.
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:217]: Maybe use a numbered list here if the order is critical?

File: release-team/role-handbooks/release-team-lead/README.md
Comment (JamesLaverack) [inline release-team/role-handbooks/release-team-lead/README.md:150]:
```suggestion
To get access to the [release team calendar](https://calendar.google.com/calendar/embed?src=kipmnllvl17vl9m98jen6ujcrs%40group.calendar.google.com) reach out to a SIG Release tech lead or chair.
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:86]: yes, good catch!

**Comment by leonardpahlke** [comment]
/unhold
ready for review

**Review by salaxander — APPROVED** [review]

File: release-team/release-team-selection.md
Comment (jrsapi) [inline release-team/release-team-selection.md:62]: When we say "should reach out" this is not a must but make an attempt? Is my understanding correct or the wording?

Comment (leonardpahlke) [reply release-team/release-team-selection.md:62]: Your understanding is correct. Its "highly suggested". We could also change this to "Has to" if we want to make this mandatory (but in that case, who's is going to check that?).

Comment (jrsapi) [reply release-team/release-team-selection.md:62]: No, not looking to make it mandatory. Thanks for the clarification.

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (sftim) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:213]: Does anyone check that the website, release blog, download page etc look right? We could have a checkbox for that.

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (sftim) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:204]: I'd mention that this is a _manual merge_ due to the freeze.

Comment (sftim) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:204]: OK to leave as-is and fix in a later PR, though.

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:192]: Oh I see. Oh well.

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:109]:
```suggestion
- [ ] Checked in with ci-signal and branch managers if 1.XX.0-alpha.2 is ok to be released and [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) tests are all green
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:103]:
```suggestion
- [ ] Ensured that there is a branch manager available for cutting 1.XX.0-alpha.1
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (JamesLaverack) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:47]:
```suggestion
    - Add an `approvers` entry in `releases/release-1.XX/OWNERS`
```

**Review by JamesLaverack — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:204]: We could link to the docs handbook for more details maybe? https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/docs/Release-Timeline.md#release-week-week-12

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:213]: agree, that is a good idea.

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (ramrodo) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:124]: I see many `release notes` words in this sentence. I think the last one should be changed to `tag`. As Release Notes Team we should generate the release notes after the tag/release for `1.XX.0-alpha.3` is generated/published.

```suggestion
- [ ] Checked in with Release Notes team on release notes progress after the tag for 1.XX.0-alpha.3 is generated
```

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (ramrodo) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:128]: Same comment as the previous one.

```suggestion
- [ ] Checked in with Release Notes team on release notes progress after the tag for 1.XX.0-alpha.4 is generated
```

**Review by ramrodo — COMMENTED** [review]
Just a clarification for the release notes

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md]: added the link ✅

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (reylejano) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:177]: non-blocking nit: I would start these conversations a week earlier
```suggestion
- [ ] Discussed Release Lead succession with the EA and sig-release leads
- [ ] Remind team lead to find successors for the upcoming cycle and discuss candidates with the EA
```

Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:177]: 👍 i will update it tomorrow

**Comment by leonardpahlke** [comment]
/hold 
For minor updates..

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:177]: The succession is mentioned before and this section here is another reminder. So I think it is fine like this.

File: .github/ISSUE_TEMPLATE/release-team-lead-progress.md
Comment (leonardpahlke) [inline .github/ISSUE_TEMPLATE/release-team-lead-progress.md:236]: do you think this needed? The next release team lead updates the team anyways

Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/release-team-lead-progress.md:236]: we can update this in a future pr too.. lets get this version merged 😅

**Comment by leonardpahlke** [comment]
it maybe a little confusing to name a column `Status` and another one `Enhancements Status`?

**Comment by puerco** [comment]
Great job, thanks Leo and everybody else adding their input!
/lgtm
/approve

**Pull request merged**
